synths/core-synths.scd

```diff
@@ -314,6 +314,30 @@ live coding them requires that you have your SuperDirt instance in an environmen
 
 		ReplaceOut.ar(out, signal);
 	}, [\ir, \ir, \ir]).add;
+	
+	// Phaser audio DSP effect definition. Params: phaserrate (Hz); phaserdepth (amplitude)
+	SynthDef("dirt_phaser" ++ numChannels, { |out, phaserrate = 1.0, phaserdepth = 0.5 |
+		var signal, phaserwave;
+		var maxDelayTime = 4;
+		var phase =  LFPar.kr(phaserrate).range(0.0088, 0.01);
+
+		~phaserBus = Bus.audio(s, 2);
+		phaserdepth = phaserdepth.clip(0, 1);
+
+		signal = In.ar(out, numChannels);
+		Out.ar(~phaserBus, signal);
+
+		// phaser algorithm #1
+		phaserwave = AllpassL.ar(In.ar(~phaserBus), maxDelayTime, phase, 0, phaserdepth);
```

Comment:
To have different phasers for different SuperDirt connections. 
For instance, d1 and d2 running two different phasers with different params values.